Adding guidelines for content reviewers

quaid opened the pull request:
- Includes critera for moving between columns in project board at https://github.com/orgs/operate-first/projects/20
- Will grow to include all that a reviewer needs to know when reviewing and approving content for the Community Handbook

Signed-off-by: Karsten Wade <kwade@redhat.com>

quaid committed:
Adding guidelines for content reviewers
- Includes critera for moving between columns in project board at https://github.com/orgs/operate-first/projects/20
- Will grow to include all that a reviewer needs to know when reviewing and approving content for the Community Handbook

Signed-off-by: Karsten Wade <kwade@redhat.com>

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **quaid** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2Fcommunity-handbook).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/operate-first/community-handbook/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["quaid"]} -->

margarethaley commented on handbook_reviewer_guidelines.md:
```diff
@@ -0,0 +1,25 @@
+== Criteria for each column to progress to the next column:
+=== To enter ‘Proposed content’
+- [ ] An issue or PR is made and assigned to the ‘Community Handbook’ project
+- [ ] To enter “Content in progress” (decider = reviewer)
```
Should line 9 be its own heading, rather than a box to check?

quaid replied:
Yep, thanks, fixed!

margarethaley commented on handbook_reviewer_guidelines.md:
```diff
@@ -0,0 +1,25 @@
+- [ ] Content meets the [style guidelines](style_guide.md)
+  - Complete has a lower standard than “complete” for user-facing documentation, as per the Community Handbook style guide.
+- [ ] Reviewer feels the content is complete/accurate and approves it
+=== To enter ‘Done’ (decider = editor/reviewer & writer)
```
Should we delete editor here in "decider = editor/reviewer & writer" - I believe we removed editor from the other lines to make things more clear

quaid replied:
Sure enough, fixed, thanks!

quaid committed:
Fixing error in the checklist

quaid committed:
Removing cruft

We had considered 'editor' as the name of the role, but it's both too-limiting and too-intimidating. Reviewer is more accurate for the goal of the Handbook, and matches the workflow.

margarethaley reviewed (approved):
/lgtm

sesheta commented:
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/operate-first/community-handbook/pull/5#pullrequestreview-703362965" title="LGTM">margarethaley</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2Fcommunity-handbook).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/operate-first/community-handbook/blob/main/OWNERS)~~ [margarethaley]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->
